xarray/tests/test_variable.py

```diff
@@ -2520,6 +2522,26 @@ def test_datetime(self):
         assert np.ndarray == type(actual)
         assert np.dtype("datetime64[ns]") == actual.dtype
 
+    @requires_pandas_version_two
+    def test_tz_datetime(self):
```

Comment:
```suggestion
    def test_tz_datetime(self) -> None:
```

Reply:
Ah indeed.  Thanks for the reminder about these!